docs/staging.adoc

```diff
@@ -0,0 +1,70 @@
+= Staging your application locally with the OpenShift registry
+
+You can prepare your application for production locally with the internal OpenShift
+Docker registry before you push your containerized application to production on OpenShift.
+
+The scenario assumes the following parameters:
+
+- OpenShift registry endpoint is *hub.openshift.adb*
+- OpenShift instance URL is *https://10.1.2.2:8443*
+- Username/password credentials is *openshift-dev/devel*
+- Application name is *my-app*
+
```

Comment:
Would it be a good idea to add ‘sample-project’ is the name of the ADB default project, in the assumed parameters section?